MovieNight/MovieNightFactory.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  MovieNightFactory.swift
+//  MovieNight
+//
+//  Created by Tassia Serrao on 13/07/2017.
+//  Copyright © 2017 Tassia Serrao. All rights reserved.
+//
+
+import Foundation
+
+class Factory {
+    
+    class func createMovieNightRepository() -> Repository {
```

Comment:
As the return type is `Repository` I think you should rename the method to `createRepository`. The fact that it is a `MovieNightRepository` is just a detail. 

Great job adding a factory btw 👏👏👏👏